commands/serve/web/connect.js

```diff
@@ -24,7 +24,7 @@ const getParams = () => {
 // QCS:       wss://<tenant-url>.<region>.qlikcloud.com/app/<app-GUID>
 // QSEoK:     wss://<host>/app/<app-GUID>
 // QSEoW:     wss://<host>/<virtual-proxy-prefix>/app/<app-GUID>
-const parseEngineURL = (url, urlRegex = /(wss?):\/\/([^/:?&]+)(?::(\d+))?/, appRegex = /\/app\/([^?&#:]+)/) => {
+const parseEngineURL = (url, urlRegex = /(wss?):\/\/([^/:?&]+)(?::(\d+))?\/?(.*)/, appRegex = /\/app\/([^?&#:]+)/) => {
```

Comment:
when testing the new urlRegex with example url above.
it looks to me that prefix will also include app. 
Could that not cause problem later in the code?

Reply:
hmm, yea true, best to strip app first

Reply:
i think the intended flow works, but the whole thing is not exactly what I thought it was. I'll redo this a bit.